types/scheduler/index.d.ts

```diff
@@ -21,6 +21,7 @@ export const unstable_IdlePriority = 5;
 export const unstable_LowPriority = 4;
 export function unstable_runWithPriority<T>(priorityLevel: number, eventHandler: () => T): T | undefined;
 export function unstable_scheduleCallback(priorityLevel: number, callback: FrameCallbackType, options?: { delay?: number, timeout?: number}): CallbackNode;
+export function unstable_next<T extends () => any>(eventHandler: T): ReturnType<T>;
```

Comment:
```suggestion
export function unstable_next<T>(eventHandler: () => T): T;
```
To bring it into conformity with how we're doing it in the rest of the file (e.g. line 22) and so we don't have to use `ReturnType`.